### PR TITLE
feat: add optional custom css

### PR DESCRIPTION
**Pull request by evolkmann**
Allows users to host their own stylesheet to customize the appearance.

- - - - - -

As this is my first interaction in this project, I want to take the chance to say thank you, it is very useful for us!

### CODE REVIEW

File: example/bee.js [hunk 1]
```diff
@@ -56,6 +56,9 @@ async function main() {
           },
         },
       ],
+
+      // Optionally include your own stylesheet
+      customCssPath: 'https://example.com/custom-arena-styles.css',
```
Comment (bradvogel) [inline example/bee.js:61]: mind removing this from the example so it doesn't 404?

File: example/bull.js [hunk 2]
```diff
@@ -54,6 +54,9 @@ async function main() {
           },
         },
       ],
+
+      // Optionally include your own stylesheet
+      customCssPath: 'https://example.com/custom-arena-styles.css',
```
Comment (bradvogel) [inline example/bull.js:59]: this too

File: README.md [hunk 3]
```diff
@@ -48,6 +48,9 @@ Arena({
       prefix: 'foo',
     },
   ],
+
+  // Optionally include your own stylesheet
+  customCssPath: 'https://example.com/custom-arena-styles.css',
```
Comment (bradvogel) [inline README.md:53]: Add to the `following optional parameters` section in the readme also.

**Review by bradvogel — CHANGES_REQUESTED** [review]
Thanks for the contribution! Just a few comments.

**Comment by evolkmann** [comment]
@bradvogel Thanks for your feedback! I agree, it should not be in the examples with that invalid URL 👍🏻

**Review by bradvogel — APPROVED** [review]

**Pull request merged**